app/workers/my_anime_list_list_worker.rb

```diff
@@ -1,19 +1,45 @@
 class MyAnimeListListWorker
   include Sidekiq::Worker
 
-  def perform(user_id)
-    kitsu_library(user_id).each do |library_entry_id|
+  def perform(linked_account_id, user_id)
+    kitsu_library(user_id).each do |library_entry|
+      # create the logs here before the next worker
+      library_entry_log = create_log(library_entry, linked_account_id)
+
       MyAnimeListSyncWorker.perform_async(
-        library_entry_id: library_entry_id,
-        method: 'create/update'
+        library_entry_id: library_entry.id,
+        # HACK: I am unable to determine the method
+        # through the initial sync because I don't access their
+        # mal list. For now everything will be a create action,
+        # but at some point we can update to get the correct action.
+        # Same issue with the LibraryEntryLog performed_action
+        method: 'create',
```

Comment:
It's not lying if the code is identical

And logically we're "updating their library" either way 🤔 I guess the verbiage isn't ideal though.  I'd rather not try and split the code at log time, since that's even more confusing than having the separate create/update codes... I guess this is fine as-is then